atrium-xrpc/src/error.rs

```diff
@@ -73,6 +73,6 @@ where
     HttpClient(Box<dyn std::error::Error + Send + Sync + 'static>),
     #[error("serde_json error: {0}")]
     SerdeJson(#[from] serde_json::Error),
-    #[error("serde_urlencoded error: {0}")]
-    SerdeUrlencoded(#[from] serde_urlencoded::ser::Error),
+    #[error("serde_qs error: {0}")]
+    SerdeQs(#[from] serde_qs::ser::Error),
```

Comment:
It seems that `serde_qs` uses `serde_qs::Error` instead of including `ser`. Can you please fix just this?https://github.com/sugyan/atrium/actions/runs/5232493995/jobs/9449498185?pr=25#step:3:261

Reply:
Ah, sorry about that - I must have only rebuilt `atrium-api`. Pushed up the change.